examples/macvtap-deviceplugin-config-default.yaml

```diff
@@ -0,0 +1,6 @@
+kind: ConfigMap
+apiVersion: v1
+metadata:
+  name: macvtap-deviceplugin-config
+data:
+  DP_MACVTAP_CONF: "[]"
```

Comment:
This is our chance to do something different here while we work on https://github.com/kubevirt/macvtap-cni/pull/10. `[]` is more akin to a blacklist while `[{"name":"*","master":"*"}]` would be more like a white-list and would allow things like `[{"name":"*","master":"*", "mode":"vepa","capacity":"50"}]` to change the default mode and capacities for the exposed links. 
Also, not as related, but if we would like to use a different term that `master` is better if we rather change it asap.

>

Reply:
> This is our chance to do something different here while we work on #10. `[]` is more akin to a blacklist while `[{"name":"*","master":"*"}]` would be more like a white-list and would allow things like `[{"name":"*","master":"*", "mode":"vepa","capacity":"50"}]` to change the default mode and capacities for the exposed links.

I'm good with your change as is, I don't have issues w/ this looking (temporarily) like a blacklist; macvtap-cni isn't even integrated into CNAO yet.

Long term, I'd prefer for it to be a whitelist, and I'm trying to come up w/ a proposal. Hopefully early next week I'll come up w/ something.

> Also, not as related, but if we would like to use a different term that `master` is better if we rather change it asap.

Yup, before this is integrated into CNAO would be good. But I'd prefer if it were done out of this PR (not related).